docs/resources/virtual_environment_vm.md

```diff
@@ -120,6 +120,8 @@ output "ubuntu_vm_public_key" {
 - `bios` - (Optional) The BIOS implementation (defaults to `seabios`).
     - `ovmf` - OVMF (UEFI).
     - `seabios` - SeaBIOS.
+- `boot_order` - (Optional) Specify the guest boot order (defaults to `c`).
```

Comment:
I think you're referring to the "legacy" parameter of the boot order which specifies a disk by its letter. We should probably switch to the device names: https://pve.proxmox.com/pve-docs/chapter-qm.html#qm_options